src/answers-umd.js

```diff
@@ -419,7 +420,11 @@ class Answers {
       onlyLegacy: true,
       onError: config.onError || function () {},
       onSuccess: config.onSuccess || function () {},
-      onFinally: config.onFinally || function () {}
+      onFinally: config.onFinally || function () {},
+      onBeforeSend: (xhr, node, url) => {
+        const uri = new URI(url).addQuery('_', new Date().getTime());
```

Comment:
could we name this variable something that hints at what it does, and maybe add a blurb to the jsdoc? I think it's hard to understand what this is for without reading your commit message